docs/docs/add-a-service-worker.md

```diff
@@ -24,7 +24,7 @@ Add this plugin to your `gatsby-config.js`
 plugins: [`gatsby-plugin-offline`]
 ```
 
-## References
+### References
```

Comment:
```suggestion
## References
```
h2 makes sense here since it provides references for the whole document. h3 would nest it under the previous section.